core/src/main/resources/hudson/triggers/SCMTrigger/config.jelly

```diff
@@ -25,7 +25,7 @@ THE SOFTWARE.
 <?jelly escape-by-default='true'?>
 <j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:d="jelly:define" xmlns:l="/lib/layout" xmlns:t="/lib/hudson" xmlns:f="/lib/form">
   <f:entry title="${%Schedule}" help="/descriptor/hudson.triggers.TimerTrigger/help/spec">
-    <f:textarea field="scmpoll_spec" checkUrl="'${rootURL}/trigger/TimerTrigger/check?value='+encodeURIComponent(this.value)"/>
+    <f:textarea field="scmpoll_spec" checkUrl="'descriptorByName/hudson.triggers.TimerTrigger/checkSpec?value='+encodeURIComponent(this.value)"/>
```

Comment:
does this not need to be /descriptorByName or use the ${rootUrl}/descriptorByName

For jobs in folders - I would expect this to be currently be an incrrect URL leading to a 404.